arangod/Agency/Supervision.cpp

```diff
@@ -1684,6 +1688,89 @@ void Supervision::cleanupFinishedAndFailedJobs() {
   cleanup(failedPrefix, maximalFailedJobs);
 }
 
+// Guarded by caller
+void arangodb::consensus::cleanupHotbackupTransferJobsFunctional(
+    Node const& snapshot,
+    std::shared_ptr<VPackBuilder> envelope) {
+  // This deletes old Hotbackup transfer jobs in 
+  // /Target/HotBackup/TransferJobs according to their time stamp.
+  // We keep at most 100 transfer jobs which are completed.
+  constexpr uint64_t maximalNumberTransferJobs = 100;
+  constexpr char const* prefix = "/Target/HotBackup/TransferJobs/";
+
+  auto const& jobs = snapshot.hasAsChildren(prefix).first;
+  if (jobs.size() <= maximalNumberTransferJobs + 6) {
```

Comment:
I think the intention behind the "+ 6" here should be explained by a code comment.

Reply:
Already added a comment.